moderngl/context.py

```diff
@@ -1307,6 +1307,7 @@ def texture_cube(
         *,
         alignment: int = 1,
         dtype: str = 'f1',
+        internal_format: int = None,
```

Comment:
Need docstring for this new keyword argument. Can probably just copy from `ctx.texture()`

Reply:
Could we just change the typing to `Optional[int]` for both cube and texture? The current typing is not correct.